### DISCUSSION TITLE
TreeTable: Frozen Rows Functionality

### DISCUSSION THREAD

**Discussion by Nickog13** [primefaces/community#2819] [PrimeReact]
Hello,

I was wondering whether it is possible to add the Frozen Rows functionality to the TreeTable analogue to, how it works for the DataTable.
I searched for alternative ways to achieve similar functionality but found, the Frozen Rows functionality from DataTable would be perfect.
Is there a specific limitation for the TreeTable preventing the usage of frozen rows?

Thanks.

**Comment by melloware** [comment]
I don't think its possible due to the nature of how the Tree is represented but I could be wrong.
